src/test/kotlin/no/item/xp/plugin/renderers/ts/RenderOptionSetFieldTest.kt

```diff
@@ -0,0 +1,84 @@
+package no.item.xp.plugin.renderers.ts
+
+import no.item.xp.plugin.models.*
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class RenderOptionSetFieldTest {
+  @Test
+  fun `render option set field`() {
+    val result = renderInterfaceModelAsTypeScript(
+      InterfaceModel(
+        "my-type",
+        listOf(
+          OptionSetField(
+            "myOptionSet",
+            "Select content manually?",
+            false,
+            false,
+            listOf(
+              ObjectField(
+                "no",
+                "No",
+                true,
+                false,
+                emptyList()
+              ),
+              ObjectField(
+                "yes",
+                "Yes",
+                true,
+                false,
+                listOf(
+                  StringField("articleList", "Select articles for the list", true, true)
+                )
+              )
+            )
+          )
+        )
+      )
+    )
+
+    assertEquals(
+      """
+      #// WARNING: This file was automatically generated by "no.item.xp.codegen". You may lose your changes if you edit it.
+      #export interface MyType {
+      #  /**
+      #   * Select content manually?
+      #   */
+      #  myOptionSet: 
+      #  | {
+      #      /**
+      #       * Selected
+      #       */
+      #      _selected: "no";
+      #
+      #      /**
+      #       * No
+      #       */
+      #      no: Record<string, unknown>
```

Comment:
The `unknown` changed to `never` right?
```suggestion
      #      no: Record<string, never>
```

Reply:
Yes, actually had that in my fork, but hadnt pushed to it 🙈